dali/pipeline/data/buffer.h

```diff
@@ -91,6 +91,21 @@ class DLL_PUBLIC Buffer {
    */
   inline Buffer() = default;
   virtual ~Buffer() = default;
+  Buffer(const Buffer &other) = delete;
+  Buffer& operator=(const Buffer &other) = delete;
```

Comment:
Isn't it automatic?

Reply:
We had `DISABLE_COPY_MOVE_ASSIGN(Buffer);` previously, so I just replicated it manually while adding the move variants.

Reply:
My point is: when we provide move variants and don't provide copy ones, they are implicitly deleted.

Reply:
I think I saw explicit deletes in the stream aware data objects PR?

Reply:
True... I wonder why I did it... Whatever then.